advanced/sidebar.md

```diff
@@ -0,0 +1,103 @@
+# Sidebar Plugin
+
+The Sidebar plugin allows you to nest your page links into interactive folders
+you can collapse and uncollapse!
```

Comment:
is this line break intentional?

Reply:
yez so it doesn't overflow in my editor, y leh? is it affecting the rendering 😟

Reply:
just checked, it doesn't break affect how it looks when rendered 👍